components/main_page/wikis/overwatch/main_page_layout_data.lua

```diff
@@ -0,0 +1,205 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:MainPageLayout/data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CONTENT = {
+	usefulArticles = {
+		heading = 'Useful Articles',
+		body = '{{Liquipedia:Useful Articles}}',
+		padding = true,
+		boxid = 1503,
+	},
+	wantToHelp = {
+		heading = 'Want To Help?',
+		body = '{{Liquipedia:Want_to_help}}',
+		padding = true,
+		boxid = 1504,
+	},
+	transfers = {
+		heading = 'Transfers',
+		body = '{{Transfer List|limit=15}}\n<div style{{=}}"display:block; text-align:center; padding:0.5em;">\n' ..
+			'<div style{{=}}"display:inline; float:left; font-style:italic;">\'\'[[#Top|Back to top]]\'\'</div>\n' ..
+			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
+			'&#91;[[Special:EditPage/Player Transfers/{{CURRENTYEAR}}/{{CURRENTMONTHNAME}}|edit]]&#93;</div>\n' ..
+			'<div style{{=}}"white-space:nowrap; display:inline; margin:0 10px font-size:15px; font-style:italic;">' ..
+			'[[Portal:Transfers|See more transfers]]<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[Transfer query]]<span style{{=}}"font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[lpcommons:Special:RunQuery/Transfer|Input Form]]' ..
+			'<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[Portal:Rumours|Rumours]]</center></div>\n</div>',
```

Comment:
Verify that this matches with what's on the page. Thought it looked likt a few of hte links have changed